### PR TITLE
WIP for reprex_scrape()

### PR DESCRIPTION
**Pull request by crew102**
Putting comments for this PR in the #67 thread

### CODE REVIEW

File: tests/testthat/test-undo.R [hunk 1]
```diff
@@ -141,3 +141,17 @@ test_that("reprex_invert() can name outfile based on input filepath", {
...
+  skip_on_cran()
+  expected <- c(
+    "#' This is a sample reprex.",
+    "# adding two vectors",
+    "x <- 1:4",
+    "y <- 2:5",
+    "x + y",
+    "#' Created on 2018-08-31 by the [reprex package](http://reprex.tidyverse.org) (v0.2.0)."
+  )
+  out <- reprex_scrape("https://github.com/r-lib/ghurl/issues/7")
```
Comment (crew102) [inline tests/testthat/test-undo.R]: Was this the use of ghurl that you had in mind?

**Comment by krlmlr** [comment]
This is what `reprex:::reprex_scrape("https://github.com/tidyverse/dplyr/issues/3826")` gets me:

```r
#' From https://github.com/tidyverse/dplyr/issues/3545#issuecomment-385150055, should we skip evaluating cases where there is no match?
library(tidyverse)
noisy_case_when <- function(..., .default = NA_character_) {
  case_when(..., TRUE ~ {
    warning("case not covered")
    .default
  })
}
noisy_case_when(
  TRUE ~ "case covered"
)
#' <sup>Created on 2018-09-15 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>
#' 
```

Amazing stuff!

Minor remarks:

- [ ] `reprex_scrape()` doesn't seem to be exported yet
- [ ] Do we want to keep the "Created on ..." ad? Do we want to add our own comment, like `<sup>Scraped on ... from ...</sup>`, instead or in addition to?
- [ ] This branch now conflicts with `master`

**Comment by crew102** [comment]
Hey @krlmlr , I was waiting until this PR was reviewed before documenting `reprex_scrape()` (though perhaps I should of at least exported it with `@export`...I'll export when doing rest of roxygen tags).

Re: how to handle the advertisement tag, I don't have a strong opinion either way.

I'll rebase on top of master when I come back to this PR after #204 is addressed.

**Comment by jennybc** [comment]
Re: keeping the ad. This is a good question. I played with "undo" some building up to this weekend's patch release. Currently, roundtrips lead to a steady accumulation of ad instances, which is gross. I think they should probably be stripped by default (?). If you are undoing and immediately calling reprex, then presence of the add suggests r-call with `advertise = TRUE`.

**Pull request closed without merging**